src/main/scala/no/ndla/articleapi/model/domain/Content.scala

```diff
@@ -84,7 +84,7 @@ object ArticleType extends Enumeration {
 case class Concept(id: Option[Long],
                    title: Seq[ConceptTitle],
                    content: Seq[ConceptContent],
-                   authors: Seq[Author],
+                   copyright: Option[Copyright],
```

Comment:
Er det for å være på linje med de andre api'ene at vi gjør copyright valgfri?
Trodde i utgangspunktet den burde vært påkrevd...

Reply:
Egentlig burde det jo ha vært påkrevd, men siden ingresser tradisjonelt sett ikke har hatt noen lisens på seg er det vel ikke mye vi kan gjøre med det?

Reply:
Antar du mener `begreper` og ikke `ingresser`?

Synes godt du kan fyre avgårde et spørsmål til Johannes. Spørre om Begrep faktisk skal ha lisens, og hvilken lisens vi i så fall skal sette på alle som ikke har det nå. (Evt. hvis det er teknisk mulig å sette det på i gammelt system, om vi skal fortsette å ha det påkrevd og melde ifra at de må gjøre endringene redaksjonelt).

Reply:
Jepp, mente begreper. Jeg spurte på forrige møte om begreper har lisens eller ikke. Fikk til svar at de ikke har hatt det, men de har skurudd på muligheten for å sette på lisens.
Jeg kan sende en mail og spørre om vi kan sette en på alle som ikke har det, eller om de har muligheten til å gjøre det på alle, selv.

Reply:
Skjønner. Men hvis de skal ha "frivillig" lisens må vi også ha en fallback i frontend som er ok for de. Tror ikke vi har det nå (alt krever lisens).

Reply:
Husker jeg rett at vi må støtte optional copyrights på begreper, slik at denne kan merges?